opentelemetry/proto/metrics/v1/metrics.proto

```diff
@@ -86,13 +90,15 @@ message InstrumentationLibraryMetrics {
 //                       |+-----+                    |
 //                       +---------------------------+
 //
-//-----------------------------------------------------------------------
-// DataPoint is a value of specific type corresponding to a given moment in
-// time. Each DataPoint is timestamped.
-//
-// DataPoint is strongly typed: each DataPoint type has a specific Protobuf message
-// depending on the value type of the metric and thus there are currently 4 DataPoint
-// messages, which correspond to the types of metric values.
+// All DataPoint types have includes three common fields:
```

Comment:
This line doesn't sound correct. 
Is that saying "All DataPoint types must include the following three common fields:"

Reply:
just removed "includes".